lib/scaffold.dart

```diff
@@ -339,9 +364,12 @@ class BackdropScaffoldState extends State<BackdropScaffold>
     // initialize _controller
     _controller = widget.controller ??
         AnimationController(
-            vsync: this, duration: Duration(milliseconds: 200), value: 1.0);
+            vsync: this, duration: Duration(milliseconds: 200), value: 1);
     if (widget.controller == null) _shouldDisposeController = true;
 
+    _backLayerScrimColorTween = _buildBackLayerScrimColorTween();
+
+    _controller.addListener(() => setState(() {}));
```

Comment:
@mockturtl Why is there a second `_controller.addListener()` needed, if we already have `_controller.addStatusListener()`?

Reply:
> has the implementation been tested against common use-cases? It should not break any previously working functionality...

I propose defining those use cases, with [test coverage](https://github.com/fluttercommunity/backdrop/blob/master/test/backdrop_test.dart#L5), as part of #5.

We can't otherwise be confident that any changes are safe.

A good starting point could be a full set of examples.

> Why is there a second _controller.addListener()

The color tween needs to update every time the [value changes](https://api.flutter.dev/flutter/animation/AnimationLocalListenersMixin/addListener.html), not just the status  ("forward / completed").

The new listener does make the `addStatusListener` redundant; I'll remove it.

Reply:
Added an example for reference.

A UX concern I notice is that users may expect tapping on the back layer scrim (red) to reveal the back layer (similar to [tapping outside a Dialog to dismiss it](https://material.io/components/dialogs#behavior)).

Reply:
About the UX concern, lets open an issue to discuss it, and keep it out of the PR for now.

Reply:
Created #78 with #5 for automatic testing with ci-cd.

Reply:
Resolving this coverstation.